install.sh

```diff
@@ -182,3 +182,14 @@ echo crew >> .git/info/sparse-checkout
 git fetch origin master
 git reset --hard origin/master
 echo "Chromebrew installed successfully and package lists updated."
+
+#check LD_LIBRARY_PATH on x86_64
+case "$architecture" in
+"x86_64")
+  (echo $LD_LIBRARY_PATH | grep '/usr/local/lib[^6]\|/usr/local/lib$' > /dev/null) || cat << EOF
```

Comment:
The problem with adding configuration in `~/.bashrc`, `~/.zshrc`, etc. is it is not easily removed when the package requiring that configuration is removed.  Also, it requires some user interaction to get things working (a la `source ~/.bashrc`).  I'm of the opinion that any configuration specific to a package should be added in a separate file that Chrome OS can read when commands requiring that specific configuration are executed.  With that said, where is the ideal location?  We don't want to edit `/etc/profile` because that would require crew to use sudo to install and remove the package.  It would be nice if Chrome OS could read configuration files from somewhere like the `/usr/local/etc/profile.d` directory.